_includes/footer.html

```diff
@@ -17,7 +17,7 @@ <h5>Mobile Services</h5>
          <li><a href="/services/push/">AeroGear <strong>Push</strong></a></li>
          <li><a href="/services/mobile-ci-cd/">AeroGear <strong>Mobile CI/CD</strong></a></li>
          <li><a href="/services/metrics/">AeroGear <strong>Metrics</strong></a></li>
-         <li><a href="/services/crc/">AeroGear <strong>CRC</strong></a></li>
+         <li><a href="/services/runtime-connector/">AeroGear <strong>Runtime Connector</strong></a></li>
```

Comment:
Do we still want to keep AeroGear in the names?

Reply:
good spot; I'll remove AG from service names to mainain consistency